packages/imperative/__tests__/plugins/cmd/install/install.handler.test.ts

```diff
@@ -50,9 +53,6 @@ import { PMFConstants } from "../../../../src/plugins/utilities/PMFConstants";
 import { TextUtils } from "../../../../../utilities";
 import { getRegistry, npmLogin } from "../../../../src/plugins/utilities/NpmFunctions";
 
-let expectedVal;
-let returnedVal;

```

Comment:
I moved the mocks to below the imports, it seems like the doMock has to be before the imports though so that had to stay there in order for the tests to pass successfully